ci-operator/step-registry/cucushift/upgrade/cucushift-upgrade-chain.yaml

```diff
@@ -8,5 +8,6 @@ chain:
   - ref: cucushift-upgrade-toimage
   - ref: cucushift-upgrade-check
   - ref: cucushift-upgrade-report
+  - ref: enable-qe-catalogsource
```

Comment:
We need do post-upgrade test based on the updated qe-app-registry, right? if so, should place it before cucushift-upgrade-check

Reply:
Following chains are also have post-upgrade test, so we needed the step too?
```
- chain: openshift-upgrade-qe-test
- chain: openshift-upgrade-qe-test-cert-rotation
- chain: openshift-upgrade-qe-test-heterogeneous
- chain: openshift-upgrade-qe-test-disconnected
- chain: openshift-upgrade-qe-test-paused-to-latest
- chain: openshift-upgrade-qe-test-paused
```

Reply:
Done

Reply:
@anpingli I only see `chain: openshift-upgrade-qe-test` updated, but not for others, or you mean no need for other upgrade chains, double check?
```
- chain: openshift-upgrade-qe-test
- chain: openshift-upgrade-qe-test-cert-rotation
- chain: openshift-upgrade-qe-test-heterogeneous
- chain: openshift-upgrade-qe-test-disconnected
- chain: openshift-upgrade-qe-test-paused-to-latest
- chain: openshift-upgrade-qe-test-paused
- chain: openshift-upgrade-qe-test-disconnected-cert-rotation
```

Reply:
It seems i didn't submit the changes.

Reply:
Done. Please refresh the page.